src/graphql/query/checkout/shopBasketUpdate.graphql

```diff
@@ -3,8 +3,9 @@
 query {
 	my {
 		userAccount {
-			promoBalance
 			id
+			balance
+			promoBalance
 		}
```

Comment:
aesthetically, i appreciate how you lined these up. 🀳

Reply:
lol, yea, I was adding and removing stuff to try and get more info there around the promo balance and just ended up matching it to the other file...